src/elements/element.line.js

```diff
@@ -22,6 +22,9 @@ function lineTo(ctx, previous, target) {
   ctx.lineTo(target.x, target.y);
 }
 
+/**
+ * @returns {any}
```

Comment:
Can't we target the 3 functions it can return here?

Reply:
@LeeLenaleee 

```
src/elements/element.line.js:86:45 - error TS2554: Expected 3-4 arguments, but got 5.

86       lineMethod(ctx, prev, point, reverse, options.stepped);
                                               ~~~~~~~~~~~~~~~

src/elements/element.line.js:94:43 - error TS2554: Expected 3-4 arguments, but got 5.

94     lineMethod(ctx, prev, point, reverse, options.stepped);
                                             ~~~~~~~~~~~~~~~


Found 2 errors in the same file, starting at: src/elements/element.line.js:86
```

It will fixed while `src/elements/element.line.js` transition to TS